cocos/2d/CCSprite.h

```diff
@@ -727,6 +738,12 @@ CC_CONSTRUCTOR_ACCESS :
 
 private:
     CC_DISALLOW_COPY_AND_ASSIGN(Sprite);
+    
+#if CC_SPRITE_DEBUG_DRAW
+    static int _debugDrawLevel;	       /// Determine to draw or not debug sprite info (0 disabled, 1 bbox, 2 texture box)
```

Comment:
It seems `_debugDrawLevel ` is not used as the description. It is used as a bool to enable/disable debug draw.

Reply:
Thanks, you are completely right, I changed to a bool to represent better the functionality, updated the interface, comments and the test